Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -435,6 +435,15 @@ class MiscellaneousTestCase: XCTestCase {
         }
     }
 
+    func testCanDebugUsinglldb() throws {
+        fixture(name: "DependencyResolution/Internal/Simple") { prefix in
+            _ = try executeSwiftBuild(prefix)
+            let exec = prefix.appending(components: ".build", "debug", "Foo").asString
+            let result = try popen(["lldb", exec, "-o", "b main.swift:2", "-o", "r", "-o", "po Bar().value", "-b"])
```

Comment:
I don't want to run lldb in our tests, can we move this to `swift-package-tests`, and have a more local check?